meilisearch/client.py

```diff
@@ -65,7 +65,7 @@ def get_indexes(self):
 
     def get_index(self, uid):
         """Get the index.
-        This index should already exist.
+        Get information about an existing index.
```

Comment:
This brings repetition with the sentence above :/ I don't really like this new change.
Maybe you can change "Get the index." into "Get information about the index." and we could let the sentence "This index should already exist." as it is?

Reply:
```suggestion
Get information about the index. This index should already exist.
```

Reply:
(Why do you remove the indentation?)
And there is still a repetition with the sentence above. Modify the sentence above instead of only modifying this one.

Reply:
Damn, nevermind. I did not see the above sentence as it is right next to `'''`. So I only red `This index should already exist`.

I'm closing this PR

Reply:
aaaaaaah! I understand better! 😄 
It happens!